glad/lang/c/loader/__init__.py

```diff
@@ -78,20 +81,23 @@
 #endif
 
 %(pre)s
-void* %(proc)s(const char *namez) {
-    void* result = NULL;
+GLADvoidfn %(proc)s(const char *namez) {
+    GLADvoidfn result = NULL;
+    void *pv = NULL;
     if(libGL == NULL) return NULL;
 
 #ifndef __APPLE__
     if(gladGetProcAddressPtr != NULL) {
-        result = gladGetProcAddressPtr(namez);
+        pv = gladGetProcAddressPtr(namez);
+        memcpy((void *)&result, (void *)&pv, sizeof(pv));
```

Comment:
When gladGetProcAddressPtr is glXGetProcAddressARB we can can to GLADvoidfn without memcpy and the extra void pointer. TODO: fix